maintainers/maintainer-list.nix

```diff
@@ -4181,6 +4181,12 @@
     githubId = 37017396;
     name = "gbtb";
   };
+  gdifolco = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.